_docs/developer-information/contribute-to-drill/020-apache-drill-contribution-ideas.md

```diff
@@ -12,9 +12,6 @@ parent: "Contribute to Drill"
     * BI Tool testing
   * General CLI improvements 
   * Eco system integrations
-    * MapReduce
-    * Hive views
-    * YARN
     * Spark
     * Hue
```

Comment:
Hue?

Reply:
Because of the reason I submitted two commits by accident, I will close this PR, and use two different PR for apache-drill-contribution-ideas.md and 050-value-vectors.md Update respectively.